models/optional_modules/core_web_vitals/snowflake/snowplow_web_vital_measurements.sql

```diff
@@ -0,0 +1,262 @@
+{{
+  config(
+    enabled=var("snowplow__enable_cwv", false) and target.type == 'snowflake' | as_bool(),
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt'))
+  )
+}}
```

Comment:
This is currently just a view right, not even a drop/create table? Depending on how intense this query is, and if this table is what will get hit by a BI tool a lot a table might be best, otherwise a view is fine.

Reply:
It is a table, perhaps I should be explicit to be clear.